src/servers/graphql-old-server.ts

```diff
@@ -374,7 +379,24 @@ const resolvers = {
 
         return status.value
       },
-      getFee: async ({ amount, invoice }) => wallet.getLightningFee({ amount, invoice }),
+      getFee: async ({ amount, invoice }) => {
+        let feeSatAmount: Satoshis | ApplicationError
+        if (amount && amount > 0) {
+          feeSatAmount = await lnNoAmountInvoiceProbeForFee({
+            amount,
+            paymentRequest: invoice,
+          })
+          if (!(feeSatAmount instanceof Error)) return feeSatAmount
```

Comment:
Yup, but it's old-server code that we'll likely be deprecating soon.

The intention is to try noAmountProbe first and if it fails with a specific error to then do amountProbe. Open to suggestions on how this can be cleaned up.